### DISCUSSION TITLE
What does `fetchFailureCount` mean in `refetchInterval`?

### DISCUSSION THREAD

**Discussion by ibrahim-ytl** [TanStack/query#7824] [General]
Someone please explain what is `fetchFailureCount` mean in `refetchInterval`? No mention about this in the docs.

**Comment by TkDodo** [comment]
`refetchInterval` gets the whole `query` passed. A query has many fields, `fetchFailureCount` is one of them. As the name hopefully suggests, it's the amount of times a fetch has failed.

**Reply by ibrahim-ytl** [reply]
Thank you for the quick response. I assume that the `fetchFailureCount` means that, something went wrong on the `useQuery` side before it can even reach my code. Am I right on this?

**Reply by TkDodo** [reply]
no? it will be incremented every time the `queryFn` returns a rejected promise

**Reply by ibrahim-ytl** [reply]
I see. What about `errorUpdatedCount`? I notice that this also increment when rejected promise is returned.
